src/unity/python/turicreate/test/test_lambda.py

```diff
@@ -76,3 +76,20 @@ def bad_fun(x):
             return x
         self.assertRaises(RuntimeError, lambda: glconnect.get_unity().parallel_eval_lambda(lambda x: bad_fun(x), ls))
         glconnect.get_unity().parallel_eval_lambda(lambda x: good_fun(x), ls)
+
+    def test_numba(self):
+
+        from numba import jit
+        import numpy as np
+        import turicreate as tc
+
+        @jit
+        def _sum(x):
```

Comment:
Probably want to call this something other than `_sum` cause it doesn't sum, even `foo` I think would be fine for a unit test.